### ISSUE TITLE
Representing module with constants

### ISSUE THREAD

**Issue by DifferentialOrange** [lunarmodules/LDoc#369]
I have the following module
```
return {
    CONST1 = 'value1',
    CONST2 = 'value2',
}
```
and I need to describe what `CONST1` and `CONST2` fields store and how they should be interpreted.

I understand that it is possible to write ldoc if my module was like that:
```
local const = {
    CONST1 = 'value1',
    CONST2 = 'value2',
}

return {
   const = const,
   -- something else
}
```
but I don't see any possibilities to describe a string (or number) field in module itself (similar to `@function` and `@table`).

**Comment by alerque** [comment]
See [Inferring more from code](https://stevedonovan.github.io/ldoc/manual/doc.md.html#Inferring_more_from_Code). I would expect this work somehting like

```lua
-- @field CONST1 foo bar
-- @field CONST2 baz qiz
local const = {
    CONST1 = 'value1',
    CONST2 = 'value2',
}

return const
```

Does that not do what you expect?

**Comment by DifferentialOrange** [comment]
> See [Inferring more from code](https://stevedonovan.github.io/ldoc/manual/doc.md.html#Inferring_more_from_Code). I would expect this work somehting like
> 
> ```lua
> -- @field CONST1 foo bar
> -- @field CONST2 baz qiz
> local const = {
>     CONST1 = 'value1',
>     CONST2 = 'value2',
> }
> 
> return const
> ```
> 
> Does that not do what you expect?

Yes, page is empty if I try this. If I mix `@table`, page is empty too.
![image](https://user-images.githubusercontent.com/20455996/149799357-11bc65b9-0750-4dc2-a43d-7986caece462.png)

I use `ldoc vs 1.4.6` and `Tarantool 2.10.0-beta2-0-g9d80b68b9` as a lua interpreter.
